node/core/pvf/src/execute/worker.rs

```diff
@@ -359,7 +366,13 @@ fn validate_using_artifact(
 		//         [`executor_intf::prepare`].
 		executor.execute(artifact_path.as_ref(), params)
 	} {
-		Err(err) => return Response::format_invalid("execute", &err),
+		Err(err) =>
+			return if err.contains("failed to open file: No such file or directory") {
+				// Raise an internal error if the file is missing.
+				Response::format_internal("missing file", &err)
+			} else {
+				Response::format_invalid("execute", &err)
```

Comment:
Before we always raised `Invalid` which would dispute. Now we raise `Internal` which does not dispute.

Reply:
Have you checked the `No such file or directory` part with a different system locale? I'm not sure how it's generated, if it relies on libc's `strerror(errno)` it could be different if the system locale differs, on my system it reads `File o directory non esistente` (because of Italian locale), but I didn't check if that's the case for this exact error message.